Description: Link against libgcc and libc to avoid unresolved symbols
 We need to link agaisnt -lgcc, on at least hppa. We also need to link
 against -lc, because we are now always using the syscall() libc function.
 So let's stop passing -nostdlib and -nostartfiles.
 .
 Note: we used to use -Wl,--as-needed, this way we made sure we pulled the
 required fortified functions from the internal libc_nonshared.a, but did
 not link against the shared library because we did not use any of its
 symbols.
Author: Guillem Jover <guillem@debian.org>
Origin: vendor
Bug-Debian: 764509
Forwarded: no
Last-Update: 2014-10-09


---
 src/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/Makefile
+++ b/src/Makefile
@@ -3,7 +3,7 @@ includedir=$(prefix)/include
 libdir=$(prefix)/lib
 
 CFLAGS ?= -g -fomit-frame-pointer -O2
-CFLAGS += -nostdlib -nostartfiles -Wall -I. -fPIC
+CFLAGS += -Wall -I. -fPIC
 SO_CFLAGS=-shared $(CFLAGS)
 L_CFLAGS=$(CFLAGS)
 LINK_FLAGS=
